src/org/ods/OdsContext.groovy

```diff
@@ -303,10 +311,50 @@ class OdsContext implements Context {
       this.environmentCreated = created
   }
 
+  boolean getCiSkipEnabled() {
+    return (config.ciSkipEnabled == null || config.ciSkipEnabled)
```

Comment:
Can we set this per default to `false`, so that we do not need to deal with `null` here? Also applies to methods further down ...

Reply:
We can set default to `true` to preserve default behavior of the ODS Pipeline.

Reply:
Ah right! Let's default to true then ...